.vscode/extensions.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
Sure you want to delete this? I do not use vscode, so I do not know if this is needed.